README.md

```diff
@@ -1,32 +1,46 @@
-# Time [![CircleCI](https://circleci.com/gh/facebook/time/tree/main.svg?style=svg)](https://circleci.com/gh/facebook/time/tree/main)
+# Time 
+
+[![CircleCI](https://circleci.com/gh/facebook/time/tree/main.svg?style=svg)](https://circleci.com/gh/facebook/time/tree/main)
+[![codecov](https://codecov.io/gh/facebook/time/branch/main/graph/badge.svg?token=QC44PEpHRi)](https://codecov.io/gh/facebook/time)
+[![Go Report Card](https://goreportcard.com/badge/github.com/leozz37/hare)](https://goreportcard.com/report/github.com/leozz37/hare)
```

Comment:
Link for different repo

Reply:
Ops, ended up forgetting to change it. Fixed!